tools/make/kube.mk

```diff
@@ -66,7 +66,7 @@ kube-demo: ## Deploy a demo backend service, gatewayclass, gateway and httproute
 	kubectl apply -f examples/kubernetes/gateway.yaml
 	kubectl apply -f examples/kubernetes/httproute.yaml
 	@echo "\nPort forward to the Envoy service using the command below"
-	@echo "kubectl -n envoy-gateway-system port-forward service/envoy-default-eg 8888:8080 &"
```

Comment:
the existing name is correct, the svc is running in the `envoy-gateway-system` system  (specified with `-n envoy-gateway-system`) with a name `envoy-<gw NS>-<gw Name>` , not the service name will soon change with https://github.com/envoyproxy/gateway/issues/441

Reply:
The gw NS is envoy-gateway-system, so the svc name should not be envoy-envoy-gateway-system-eg?

Reply:
the gw NS here is the namespace where the gateway resource (`gateway.networking.k8s.io/v1beta1`) resides in / created in, not where the envoy gateway deployment/svc is running in